Tools/refcounts/.ruff.toml

```diff
@@ -0,0 +1,34 @@
+target-version = "py312"
+line-length = 79
+fix = true
+
+[format]
+quote-style = "single"
```

Comment:
don't tell the Black maintainers on the team

Reply:
I don't like double quotes because I need TWO keystrokes! (shift+2) but for single quotes I can do it in one!

Reply:
But actually, maybe it's better to have a double quote-style to be consistent with the rest of the project.